gradio/helpers.py

```diff
@@ -353,8 +353,7 @@ def _start_caching(self):
                 warnings.warn(
                     "Setting `cache_examples=True` is not supported in the Wasm mode. You can set `cache_examples='lazy'` to cache examples after first use."
                 )
-            else:
-                client_utils.synchronize_async(self.cache)
+            await self.cache()
 
     def lazy_cache(self) -> None:
```

Comment:
I think `lazy_cache` can also be made a co-routine?

Reply:
Done